assets/js/index.js

```diff
@@ -0,0 +1,296 @@
+const API_KEY = "7ec1ea2463d21d115915eb7b42565bed";
+
+const searchHistoryContainer = $("#search-history-container");
+
+const searchBtn = $("#search-button");
+
+const form = $("#select-city");
+
+const formInput = $("#input-city");
+
+const searchHistory = $("#search-history");
+
+const cityButton = $("#city-button");
+
+const weatherContainer = $("#weather-container");
+
+const currentWeather = $(".current-weather");
+
+const forecastContainer = $("#forecast-container");
+
+const currentDate = moment.unix().format("dddd Do MMMM YYYY");
+
+// initialise LS
+const onReady = () => {
+  // check if cities exists in LS
+  const recentCities = readFromLocalStorage();
+  // if false then set tasks to empty object in LS
+  if (!recentCities) {
+    localStorage.setItem("cities", JSON.stringify([]));
+  }
+
+  renderCities();
+};
+
+// get cities from local storage
+const readFromLocalStorage = () => {
+  // get from LS by key
+  const getCities = localStorage.getItem("cities");
+  // parse LS data
+  return JSON.parse(getCities);
+};
+
+const renderCities = () => {
+  // get recent cities from LS []
+  const recentCities = readFromLocalStorage();
+
+  const message = "Your 6 most recent searches will be saved here.";
+
+  // if [] is empty then render alert
+  if (!recentCities.length) {
+    searchHistory.append(`<h2 class="message">${message}</h2>`);
+  } else {
+    // else render all recent cities
+
+    for (let i = 0; i < 6; i += 1) {
+      if (!recentCities[i]) {
```

Comment:
It's a bit odd to have an empty `if` block like this. You could just switch the condition around and do

```
if (recentCities[i])
```

and then you wouldn't need an `else` block